packages/web/src/components/edit/fields/visibility/VisibilityField.tsx

```diff
@@ -30,14 +30,16 @@ import { mergeReleaseDateValues } from './mergeReleaseDateValues'
 
 const messages = {
   ...visibilityMessages,
-  scheduled: (date: string) => `Scheduled for ${formatCalendarTime(date)}`
+  scheduled: (date: string) => `Scheduled for ${formatCalendarTime(date)}`,
+  emptyPlaylistTooltipText: 'You must add at least 1 song.'
```

Comment:
does it have to be a public song?

Reply:
I guess based on how we do the checks, no, but maybe we can address that separately? good thought...!